src/components/Employee/Taxes/Taxes.tsx

```diff
@@ -72,6 +72,7 @@ const Root = (props: TaxesProps) => {
 
   const defaultValues = {
     ...employeeFederalTax,
+    filingStatus: employeeFederalTax.filingStatus ?? undefined,
```

Comment:
Msc type error introduced presumably by bumping the speakeasy version. Just does a conversion so filingStatus is ensured to be `string | undefined` instead of `string | null` which was causing default values to error out here